lib/stronger_parameters/constraints/null_string_contraint.rb

```diff
@@ -0,0 +1,17 @@
+require 'stronger_parameters/constraints'
+
+module StrongerParameters
+  class NullStringConstraint < Constraint
+    NULL_VALUES = ['', 'undefined'].freeze
+
+    def value(v)
+      if v.nil?
```

Comment:
<-> "NullString" sounds like a String to me, maybe `JavascriptNull` or something like that ?